src/Microsoft.HttpRepl/Program.cs

```diff
@@ -22,18 +23,25 @@ public class Program
     {
         public static async Task Main(string[] args)
         {
-            await new Program().Start(args);
+            await Start(args);
         }
 
-        public async Task Start(string[] args, IConsoleManager consoleManager = null, IPreferences preferences = null)
+        internal static async Task Start(string[] args, IConsoleManager consoleManager = null, IPreferences preferences = null, ITelemetry telemetry = null)
```

Comment:
Why the change to internal?  This was left public to force the IntegrationTests to not have an IVT, so that they wouldn't dabble with internal types.

Reply:
It was because code analysis complained about not checking `consoleManager`, `preferences` and `telemetry` for `null` before using them, since they were in the public interface. Setting aside that they _were_ checked before use, in `ComposeDependencies`, it wasn't _really_ a public interface in my mind, so I changed it to `internal`. But I didn't really like having the IVT either. So I changed it back and added the suppression attribute instead. Hopefully I can get that analysis warning to go away once I add nullable annotations (though my quick try didn't work - code analysis didn't seem to care about the `[NotNull]` attributes on `ComposeDependencies` parameters).